Source/UI/NewUI/BillingForm/Factory/BillingFormFactory.swift

```diff
@@ -18,7 +18,7 @@ public struct BillingFormFactory {
          .phoneNumber(DefaultBillingFormPhoneNumberCellStyle())]
     }
 
-    static func getBillingFormViewController(style: BillingFormStyle?, data: BillingForm?, delegate: BillingFormViewModelDelegate) -> (BillingFormViewModelDelegate?, UINavigationController)? {
+    static func getBillingFormViewController(style: BillingFormStyle?, data: BillingForm?, delegate: BillingFormViewModelDelegate) -> (deleagte: BillingFormViewModelDelegate?, navigationController: UINavigationController)? {
```

Comment:
```suggestion
    static func getBillingFormViewController(style: BillingFormStyle?, data: BillingForm?, delegate: BillingFormViewModelDelegate) -> (delegate: BillingFormViewModelDelegate?, navigationController: UINavigationController)? {
```

Reply:
I'm late for this, but why are we passing in a delegate and then also giving it back out. Doesn't the caller already own it since it provided it in the first place?

Reply:
Agree with @alex-ioja-yang-cko , looking at the code i see getBillingFormViewController takes in a delegate which is of the view controller where the billingVC is presented. It is then passed to DefaultBillingFormViewModel where it has the delegate methods and sets this delegate. 

But from the sample app when we call getBillingFormViewController as merchant it is never used inside the VC itself ( when returned ) and we can get rid of the returned delegate if we are not using it.

Reply:
we should address this and/or fix the typo

Reply:
It was a comment on old PR to expose [delegate](https://github.com/checkout/frames-ios/pull/144#discussion_r860643734)

Reply:
I think you misinterpreted what that comment asked for. It wasn't for a delegate to be returned alongside the VC, it wouldn't make sense to if you have no need for it.

I was saying that the common syntax is to assign the delegate **after** the object has been created, not during the initialisation. The common way to see delegate being assigned is in patterns of:
```swift
let object = MyObject()
object.delegate = myDelegate
```

rather than

```swift
let object = MyObject(delegate: myDelegate)
```

There are some nuances to why that is, which are not applicable here, but make it a more regular occurance when navigating the code. I would have rather you rejected this idea than add this code change :).